module-2/backend/week_two.md

```diff
@@ -12,70 +13,99 @@ end
 
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
 
-  we can call Team.all to find all the elements for Team table
-  We can call Team.create(attributes) to create a Team object.
-**activerecord has all those methods, we can call them on Team as Team is inherited
-from Activerecord: :Base
+  * we can call Team.all to find all the elements for Team table
+  * We can call Team.create(attributes) to create a Team object.
+
+  * activerecord has all those methods, we can call them on Team as Team is inherited
+  from Activerecord: :Base
 
 3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
 
-  team = Team.find(:id)
-  team.name
+  * team = Team.find(:id)
+  * team.name
 
-  owner = Owner.find(team.owner_id)
+  * owner = Owner.find(team.owner_id)
 
 4. Assume that you added a line to your `Team` class as follows:
 
-class Team << ActiveRecord::Base
-  belongs_to :owner
-end
+  class Team << ActiveRecord::Base
+    belongs_to :owner
+  end
 
 Now how would you find the owner of the team with an id of 4?
-  team = Team.find(:id)
-  owner = team.owner
+
+  * team = Team.find(:id)
+
+  * owner = team.owner
```

Comment:
could also do `owner = Team.find(4).owner` now that you have the `belongs_to` relationship